p2p_wallet/AppCoordinator/AppCoordinator.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  AppCoordinator.swift
+//  p2p_wallet
+//
+//  Created by Chung Tran on 23/05/2022.
+//
+
+import AnalyticsManager
+import Foundation
+import Resolver
+import SolanaSwift
+import UIKit
+
+class AppCoordinator {
+    // MARK: - Dependencies
+
+    private var appEventHandler: AppEventHandlerType = Resolver.resolve()
+    private let storage: AccountStorageType & PincodeStorageType & NameStorageType = Resolver.resolve()
+    let analyticsManager: AnalyticsManager = Resolver.resolve()
+    let notificationsService: NotificationService = Resolver.resolve()
+
+    // MARK: - Properties
+
+    let window: UIWindow
+    let lockView = LockView()
```

Comment:
Do you need to contain this in memory? I think you need to make LockView as optional and when you remove it, set it to nil also.